src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -565,6 +563,8 @@ private static SymbolDisplayPartKind GetPartKind(INamedTypeSymbol symbol)
             {
                 case TypeKind.Submission:
                 case TypeKind.Module:
+                case TypeKind.Class when symbol.IsRecord:
+                    return SymbolDisplayPartKind.RecordName;
```

Comment:
>return SymbolDisplayPartKind.RecordName; [](start = 20, length = 40)

This doesn't look right. Submission and Module should remain SymbolDisplayPartKind.ClassName

Reply:
Ah, my bad. Thanks for the catch - fixed.